### DISCUSSION TITLE
Change organization name

### DISCUSSION THREAD

**Discussion by miguerizzo** [test-results-reporter/reporter#241] [Q&A] [answered]
Hey all, It is possible to change organization name?

**Comment by ASaiAnudeep** [comment, marked as answer]
Hello @miguerizzo , the capability to update org details would be available by this week. I will update this thread once the changes are released.

**Reply by ASaiAnudeep** [reply]
Changes are live now. Please click on the edit icon next to the header to change the name of your org.
